.pipelines/azure_pipeline_mergedbranches.yaml

```diff
@@ -27,12 +27,11 @@ jobs:
     name: Azure-Pipelines-CI-Test-EO
   steps:
   - task: ComponentGovernanceComponentDetection@0
-
+  ## commit will be in format 3.0.0-14-g833be407
```

Comment:
where do you specify that commit id has to be in this format "3.0.0-14-g833be407"?

Reply:
If the release tag we set are the semver format like x.x.x, then the   'git describe' command will return this format